nvflare/private/fed/app/client/worker_process.py

```diff
@@ -88,17 +89,17 @@ def main():
                 )
             )
 
-        workspace = os.path.join("/tmp/fl", client_name)
+        startup = args.startup
```

Comment:
What's the difference btw this startup and the startup on line 75? If they are the same, then we shouldn't need this new arg for startup.